api/onboardingpb/json/onboardingpb.json

```diff
@@ -0,0 +1,192 @@
+                  "x-order": 1
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/v1/onboarding/tips/{tipId}/type/{tipType}/user/{userId}": {
+      "get": {
+        "description": "Retrieve tip details from PMM server",
+        "tags": [
+          "TipService"
+        ],
+        "summary": "Get tip details",
+        "operationId": "GetTipStatus",
+        "parameters": [
+          {
+            "type": "integer",
+            "format": "int32",
+            "description": "The ID of the tip to retrieve.",
+            "name": "tipId",
+            "in": "path",
+            "required": true
+          },
+          {
+            "enum": [
+              "SYSTEM",
+              "USER"
+            ],
+            "type": "string",
+            "description": "Tip Type (system or user)",
+            "name": "tipType",
+            "in": "path",
+            "required": true
+          },
+          {
+            "type": "integer",
+            "format": "int32",
+            "description": "The User ID.",
+            "name": "userId",
+            "in": "path",
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "A successful response.",
+            "schema": {
+              "description": "Response message for GetTipResponse method.",
+              "type": "object",
+              "properties": {
+                "isCompleted": {
+                  "description": "Show whether tip is completed or not.",
+                  "type": "boolean",
+                  "x-order": 1
+                },
+                "tipId": {
+                  "description": "The ID of the tip to retrieve.",
+                  "type": "integer",
+                  "format": "int32",
+                  "x-order": 0
+                }
+              }
+            }
+          },
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "code": {
+                  "type": "integer",
+                  "format": "int32",
+                  "x-order": 0
+                },
+                "details": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "@type": {
+                        "type": "string",
+                        "x-order": 0
+                      }
+                    },
+                    "additionalProperties": false
+                  },
+                  "x-order": 2
+                },
+                "message": {
+                  "type": "string",
+                  "x-order": 1
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "tags": [
+    {
+      "name": "TipService"
+    }
+  ]
+}
```

Comment:
please add last empty line

Reply:
it's generated automatically, if I modify it someone else will update it again.